keyboards/druah/efreet/info.json

```diff
@@ -0,0 +1,87 @@
+        }
+    },
+    "url": "https://druah.moe",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x1337",
+        "vid": "0x444E"
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x12": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
+                {"matrix": [0, 1], "x": 1, "y": 0},
+                {"matrix": [0, 2], "x": 1.99, "y": 0},
+                {"matrix": [0, 3], "x": 2.99, "y": 0},
+                {"matrix": [0, 4], "x": 3.99, "y": 0},
+                {"matrix": [0, 5], "x": 4.99, "y": 0},
+                {"matrix": [0, 6], "x": 5.98, "y": 0},
+                {"matrix": [0, 7], "x": 6.98, "y": 0},
+                {"matrix": [0, 8], "x": 7.98, "y": 0},
+                {"matrix": [0, 9], "x": 8.98, "y": 0},
+                {"matrix": [0, 10], "x": 9.97, "y": 0},
+                {"matrix": [0, 11], "x": 10.97, "y": 0},
+                {"matrix": [1, 0], "x": 0, "y": 1},
+                {"matrix": [1, 1], "x": 1, "y": 1},
+                {"matrix": [1, 2], "x": 1.99, "y": 1},
+                {"matrix": [1, 3], "x": 2.99, "y": 1},
+                {"matrix": [1, 4], "x": 3.99, "y": 1},
+                {"matrix": [1, 5], "x": 4.99, "y": 1},
+                {"matrix": [1, 6], "x": 5.98, "y": 1},
+                {"matrix": [1, 7], "x": 6.98, "y": 1},
+                {"matrix": [1, 8], "x": 7.98, "y": 1},
+                {"matrix": [1, 9], "x": 8.98, "y": 1},
+                {"matrix": [1, 10], "x": 9.97, "y": 1},
+                {"matrix": [1, 11], "x": 10.97, "y": 1},
+                {"matrix": [2, 0], "x": 0, "y": 1.99},
+                {"matrix": [2, 1], "x": 1, "y": 1.99},
+                {"matrix": [2, 2], "x": 1.99, "y": 1.99},
+                {"matrix": [2, 3], "x": 2.99, "y": 1.99},
+                {"matrix": [2, 4], "x": 3.99, "y": 1.99},
+                {"matrix": [2, 5], "x": 4.99, "y": 1.99},
+                {"matrix": [2, 6], "x": 5.98, "y": 1.99},
+                {"matrix": [2, 7], "x": 6.98, "y": 1.99},
+                {"matrix": [2, 8], "x": 7.98, "y": 1.99},
+                {"matrix": [2, 9], "x": 8.98, "y": 1.99},
+                {"matrix": [2, 10], "x": 9.97, "y": 1.99},
+                {"matrix": [2, 11], "x": 10.97, "y": 1.99},
+                {"matrix": [3, 0], "x": 0, "y": 2.99},
+                {"matrix": [3, 1], "x": 1, "y": 2.99},
+                {"matrix": [3, 2], "x": 1.99, "y": 2.99},
+                {"matrix": [3, 3], "x": 2.99, "y": 2.99},
+                {"matrix": [3, 4], "x": 3.99, "y": 2.99},
+                {"matrix": [3, 5], "x": 4.99, "y": 2.99},
+                {"matrix": [3, 6], "x": 5.98, "y": 2.99},
+                {"matrix": [3, 7], "x": 6.98, "y": 2.99},
+                {"matrix": [3, 8], "x": 7.98, "y": 2.99},
+                {"matrix": [3, 9], "x": 8.98, "y": 2.99},
+                {"matrix": [3, 10], "x": 9.97, "y": 2.99},
+                {"matrix": [3, 11], "x": 10.97, "y": 2.99}
```

Comment:
```suggestion
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1},
                {"matrix": [1, 1], "x": 1, "y": 1},
                {"matrix": [1, 2], "x": 2, "y": 1},
                {"matrix": [1, 3], "x": 3, "y": 1},
                {"matrix": [1, 4], "x": 4, "y": 1},
                {"matrix": [1, 5], "x": 5, "y": 1},
                {"matrix": [1, 6], "x": 6, "y": 1},
                {"matrix": [1, 7], "x": 7, "y": 1},
                {"matrix": [1, 8], "x": 8, "y": 1},
                {"matrix": [1, 9], "x": 9, "y": 1},
                {"matrix": [1, 10], "x": 10, "y": 1},
                {"matrix": [1, 11], "x": 11, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2},
                {"matrix": [2, 1], "x": 1, "y": 2},
                {"matrix": [2, 2], "x": 2, "y": 2},
                {"matrix": [2, 3], "x": 3, "y": 2},
                {"matrix": [2, 4], "x": 4, "y": 2},
                {"matrix": [2, 5], "x": 5, "y": 2},
                {"matrix": [2, 6], "x": 6, "y": 2},
                {"matrix": [2, 7], "x": 7, "y": 2},
                {"matrix": [2, 8], "x": 8, "y": 2},
                {"matrix": [2, 9], "x": 9, "y": 2},
                {"matrix": [2, 10], "x": 10, "y": 2},
                {"matrix": [2, 11], "x": 11, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3},
                {"matrix": [3, 1], "x": 1, "y": 3},
                {"matrix": [3, 2], "x": 2, "y": 3},
                {"matrix": [3, 3], "x": 3, "y": 3},
                {"matrix": [3, 4], "x": 4, "y": 3},
                {"matrix": [3, 5], "x": 5, "y": 3},
                {"matrix": [3, 6], "x": 6, "y": 3},
                {"matrix": [3, 7], "x": 7, "y": 3},
                {"matrix": [3, 8], "x": 8, "y": 3},
                {"matrix": [3, 9], "x": 9, "y": 3},
                {"matrix": [3, 10], "x": 10, "y": 3},
                {"matrix": [3, 11], "x": 11, "y": 3}
```

Reply:
> 

This PCB uses 19mm switch spacing instead of the standard 19.05, hence why the X and Y values aren't whole numbers. Should I still change them to whole numbers like suggested?

Reply:
`info.json` doesnt really care about spacing as its built on the assumption of "key units".

Having decimal positions, with the default height and width as 1, is just going to cause display issues in stuff like <https://config.qmk.fm/>.